.github/CODEOWNERS

```diff
@@ -0,0 +1,34 @@
+# github folder and all project root files (readme, license, general config, etc.)
+.github/      @bartlomieju @kt3k
+/*            @bartlomieju @kt3k
```

Comment:
This glob specifically matches all files located on project root (like `deno.json`, `LICENSE`, etc.)